app/controllers/admin/feedback_messages_controller.rb

```diff
@@ -107,7 +107,7 @@ def generate_message(params)
       <<~HEREDOC
```

Comment:
I just noticed that this method might not be called anywhere...

Reply:
@rhymes nice catch! I've made a note and I'll add this as a separate PR so that this PR stays focussed.